src/main/java/codeu/model/data/User.java

```diff
@@ -88,6 +87,5 @@ public void setDescription(String Description){
   /** Allows to know if the user is an admin */
   public Boolean isAdmin() {
     return (name.equals("Ricardo")||name.equals("Manjil")|| name.equals("Kirielle"));
```

Comment:
Can you add Tofe as well?

Reply:
Ricardo has already changed this admin stuff you probably dont need to.

Reply:
Got it!